scripts/validationAndPreparation.sh

```diff
@@ -1,45 +1,82 @@
+#!/bin/bash
+alias connect_server="sftp -i /root/.ssh/id_rsa_worker validationworker@trusted-setup.staging.gnosisdev.com"
```

Comment:
Might be good to not hardcode the URL anywhere but instead supply is as an ENV var in the docker setup?